src/core/Global.ttslua

```diff
@@ -952,3 +957,26 @@ function onClick_defaultSettings()
   -- update UI
   updateOptionPanelState()
 end
+
+-- splash scenario title on setup
+titleSplashId = nil
+function titleSplash(params)
```

Comment:
Will there be any parameters besides `currentScenario`? If not, I'd change the argument to not be a table.